### DISCUSSION TITLE
Custom validation message does not seem to work in version 3.4.10

### DISCUSSION THREAD

**Discussion by faltjo** [livewire/livewire#8292] [7. Bugs (archived)] [answered]
### Livewire version

v3.4.10

### Laravel version

v10.34.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Create a new component with a property which uses `Validate`:

```
namespace App\Livewire;

use Livewire\Attributes\Validate;
use Livewire\Component;

#[Validate('required|email', message: 'custom message', onUpdate: false)]
public string $email;

public function render()
{
    return view('site.livewire.test-validation');
}

public function submit()
{
    $this->validate();
}
```

2. In the corresponding view create a form, which uses the `submit` method on submit:
```
<div>
    <form wire:submit="submit">
        <div>
            <label for="email">E-Mail</label>
            <input type="text" name="email" id="email" required wire:model="email">
            @error('email')
                <div>
                    <p>{{ $message }}</p>
                </div>
            @enderror
        </div>
        <div>
            <span>
                <button type="submit">SUBMIT</button>
            </span>
        </div>
    </form>
</div>
```
3. Browse welcome page and enter some invalid string into the input field and submit the form. Instead of the expected `custom message`, the message `The email must be a valid email address.` is displayed as error.

Additional info:
I also tried this without the `onUpdate` parameter.
I had to move the generated livewire view to `resources/views/sites/livewire` instead of `resources/views/livewire`.

### Repository or Wirebox URL

https://github.com/faltjo/livewire-validation

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by danie-ramdhani** [comment, marked as answer]
it's not bug. please https://livewire.laravel.com/docs/validation#custom-messages

you're using two rules here (`required|email`), so the validate message should be look like this

```php
#[Validate('required|email', message: [
    'required' => 'Your required message',
    'email' => 'Your email message',
])]
public string $email;
```

**Reply by faltjo** [reply]
Thanks for pointing out to me.
